yo/db.py

```diff
@@ -300,16 +303,71 @@ def get_priority_count(self,
                 logger.exception('Exception occurred!')
         return retval
 
+    def create_wwwpoll_notification(self,
+                                    notify_id=None,
+                                    notify_type=None,
+                                    created_time=None,
+                                    to_user=None,
+                                    raw_data={}):
```

Comment:
```python
In[1]: def func(raw_data={},**kwargs):
...:         raw_data.update(kwargs)
...:         print(raw_data)
In[2]: func(a=1,b=2)
{'a': 1, 'b': 2}
In[3]: func(c=3,d=4)
{'a': 1, 'b': 2, 'c': 3, 'd': 4}

```

Reply:
Not following you here? raw_data is distinct from the other params